### ISSUE TITLE
add project-level metadata to export options

### ISSUE THREAD

**Issue by timothytsai** [OuhscBbmc/REDCapR#236]
Hi,

It would be useful to be able to export project-level metadata, specifically project ID.  Our use case is creating links to records in conjunction with the REDCap version number (already exportable). 

I have a working implementation at my fork (https://github.com/timothytsai/REDCapR/commit/9a7bf004f9151e292c93731460d7c0b4c1ad3b5f), which I can convert to a pull request if desired.

Cheers,
Tim

**Comment by timothytsai** [comment]
Hi @wibeasley , I'd love to give it a shot.  I will plan to add this as a new function, returning a list with appropriate data types.  I'll do some research into matching the data types to the types in the backend.

I'll try to emulate the existing formatting of your code as best I can, but do please let me know if there are any specific coding style conventions that I should take note of.

**Comment by pbchase** [comment]
I started writing this yesterday as a hack of redcap_metadata_read before I found this issue. The working title of the draft function is `redcap_project_info_read`. Our use-case is to verify a token belongs to a project by loading the token and a PID via the same `.env` file, connecting, reading the project PID from the server, and testing for a match. 

What's the status on your work, @timothytsai?

**Comment by timothytsai** [comment]
Hi @pbchase , I am hoping to pick this up again this month.  My fork at https://github.com/timothytsai/REDCapR/commit/9a7bf004f9151e292c93731460d7c0b4c1ad3b5f is the modification of the existing function, which should be migrated to its own function as it sounds like you have started.

**Comment by wibeasley** [comment]
@timothytsai, sorry the timing didn't work at REDCapCon to catch up.  I was hoping to chat.

Unrelated, I was reviewing the older issues in this repo and realized that @skadauke's suggestion in #410.  I wished I figured it out sooner, so I wouldn't have duplicated some of your work from 2018.  Start a new issue and reference this one if youthink there's something missing (from the function that was added last month).

* https://ouhscbbmc.github.io/REDCapR/reference/redcap_project_info_read.html
* https://github.com/OuhscBbmc/REDCapR/blob/HEAD/R/redcap-project-info-read.R
